### PR TITLE
Maistra 2.3.2 2

### PR DESCRIPTION
**Pull request by twghu**
Update on PR 225 to address use of cherry-picks.

### CODE REVIEW

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -1037,14 +1026,7 @@ def _proxy_wasm_cpp_sdk():
     external_http_archive(name = "proxy_wasm_cpp_sdk")
 
 def _proxy_wasm_cpp_host():
-    external_http_archive(
-        name = "proxy_wasm_cpp_host",
-        # proxy-wasm-cpp-host-s390x-support.patch fixes WASM on s390x error https://issues.redhat.com/browse/OSSM-1815
-        # maistra-2.3 issue: https://issues.redhat.com/browse/OSSM-1956
-        # The permanent fix is: https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/282
-        patches = ["@envoy//bazel/external:proxy-wasm-cpp-host-s390x-support.patch"],
-        patch_args = ["-p1"],
```
Comment (jwendell) [inline bazel/repositories.bzl:1046]: Are you sure about this? To me this was the most important change I commented in PR 225. The others were most likely just nit's.

Comment (twghu) [reply bazel/repositories.bzl:1046]: Yes, I discussed this with Konstantin.

The dependency has been updated upstream, and Konstantin indicates the patch he needs is in place.

Comment (jwendell) [reply bazel/repositories.bzl:1046]: Great. Then I'm assuming the upstream changes are present in `624ef2edac2cdc086e0339f36b32ef1a2ed7e270` which is the commit we are pulling in in repository_locations.bzl, right?

cc @knm3000

Comment (knm3000) [reply bazel/repositories.bzl:1046]: correct, https://github.com/proxy-wasm/proxy-wasm-cpp-host/commits/624ef2edac2cdc086e0339f36b32ef1a2ed7e270 includes PR 282 "Do not reverse bytes for NullVM":
```
Commits on Oct 2, 2022
Do not reverse bytes for NullVM (#282).
knm3000 committed on Oct 2, 2022
```

Comment (jwendell) [reply bazel/repositories.bzl:1046]: Awesome! Thanks for checking this @knm3000. And sorry for the noise @twghu

**Comment by jwendell** [comment]
@twghu It looks like there are newer commits on Envoy 1.22 (1.22.8 tag). Do you want to bring them too?

**Review by tedjpoole — APPROVED** [review]
LGTM

**Pull request merged**